makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: install-devel test lint pytest check-clean-tree fix force-fix venv
+
+# Customize your venv name by running make as "ZT_VENV=my_venv_name make <command>"
```

Comment:
We should mention that the `ZT_VENV=...` would not only be required for the first time but every time it is run.

Additionally, perhaps rename `<command>` to `<zt-command>`?

Also, should we rather include this in the README so that it is easier to find?

Reply:
I had specifically left these notes in the makefile for now as they seemed more 'advanced', and may be scope for further extension - they work, but for now it's easier to just have one environment with a standard name.

I'm not sure we gain from `zt-command` - what matters is running the different makefile 'targets', which are more just specific to those in the makefile.